.changeset/selfish-ladybugs-smile.md

```diff
@@ -0,0 +1,6 @@
+---
+'@clerk/clerk-js': minor
+'@clerk/types': minor
+---
+
+Deprecate `afterSignOutUrl` from UserButton.
```

Comment:
Can you specify what actions a user needs to take here w.r.t. the deprecation? What do they also need to change in their code?

Reply:
makes sense, i updated it